content/release/ReleaseNotes345.md

```diff
@@ -0,0 +1,57 @@
+<!--
+title: "Contrast 3.4.5 - October 2017"
+description: "Contrast 3.4.5 October 2017"
+tags: "3.4.5 October Release Notes"
+-->
+
+Remediate old vulnerabilities, tag from any agent, and apply and Assess Rules to new applications - it's automatic. 
+
+## Fixes
+
+* Admins retain role permissions when managing Application Exclusions. 
+* Assess Rules are consistent with vulnerability details. 
+* Views of the Attack Trend and Attack Events are more accurate. 
+* Integration testing for Serena Business Management is successful. 
+* Tooltips in Vulnerability Trend chart show up at the right time. 
+* Work item descriptions are correct for Visual Studio Team Servers two-way integration.
+* License Ruby applications from the Applications page.
+* Feature guarding applies correctly to Node.
+* Labels for source events are complete.
+* Super and System Admins can view vulnerabilities without errors.
+* FixLibrariesData Task is successful again.
+* Cold Fusion support is restored.
+* Vulnerability severity changes stop when you cancel them.
+* Parameters in vulnerability overviews display correctly. 
+* Unwanted email notifications for server updates and Remediation Policy are over. 
+* SaaS users can navigate to Contrast in additional browser tabs and access API documentation. 
+
+## Improvements 
+
+* Do yourself a favor, and create default settings for new applications in your organization. Just go to the new **Applications** tab in Organization Settings, and set up [Application Defaults](admin-orgsettings.html#app-defaults) based on levels of importance, existing policies and Assess licensing.
+
+* Remediation Policies have a new outcome: [automatically remediate a vulnerability](admin-policymgmt.html#remediate). After you select this option, Contrast will change the vulnerability status to “Auto-Remediated”, if it hasn’t rediscovered the vulnerability by your given deadline. 
+
+* Tag vulnerabilities, applications, servers or libraries automatically from any agent - just like you wanted. (Hint: This practice is especially helpful when multiple groups are working within Contrast and want to quickly filter without taking the time for manual tagging.)
+
+* Manage [Assess Rules](admin-policymgmt.html#assess) per application to customize your security requirements. You can also set a default policy that will apply to newly onboarded applications. 
+
+* Set our .NET agents to a pending update state when a new agent is available. In this state, .NET agents will wait for users to restart their IIS servers before installing an update.
```

Comment:
Should this be in the .Net Summary?  I don't have a strong opinion, just checking.

Reply:
Product included their "What's New" list of highlights, so I kept it in this "Improvements" section - but, I'm open to moving it. 

@DanFiedler - Do you have a preference?